docs/UserGuide.md

```diff
@@ -75,10 +75,24 @@ use via a CLI while still having the benefits of a GUI.
     - If a parameter is expected only once in the command, but you specified it multiple times,
       an error message will be shown.
       
-    - If an extraneous parameter is specified, an error message will be shown.
+    - If an extraneous parameter is specified, an error message will be shown. (Does not apply for tags or contacts)
+    
+    - Parameter or argument values cannot contain the character `/`. An error message will be shown if `/` is present in the argument value. For example, `task add 1/2` is invalid, and can be replaced with valid alternatives such as `task add 1 or 2` instead.
 
 </div>
 
+### Notable command prefix formats
+
+Some prefixes require the user to follow certain formats in their commands.
+
+| **Prefix**  | **Use** | **Format** |
+| ------------- | ------------- | ------------- |
+| `ts`  | Timestamp, represents a date  | `DD`-`MM`-`YYYY` <br/><br/> 1. Single-digit dates or months require a `0` appended at the front. For example, 3rd July 2021 is represented as `03-07-2021`. <br/><br/> 2. The day, month and year are separated by hyphens. |
+| `e`  | Email  | `local-part@domain` <br/><br/> 1. The local-part should only contain alphanumeric characters and these special characters, excluding the parentheses, (+_.-). The local-part may not start or end with any special characters.<br/><br/>2. This is followed by a '@' and then a domain name. The domain name is made up of domain labels separated by periods. <br/><br/> The domain name must: <br/>     - end with a domain label at least 2 characters long <br/> - have each domain label start and end with alphanumeric characters <br/> - have each domain label consist of alphanumeric characters, separated only by hyphens, if any. |
+
+
+<a name="ab3comm"></a>
```

Comment:
What's this for?

Reply:
Email format provided by the AB3 itself. 
To fix #110 , think i forgot to link it

Reply:
But what is this for?
`<a name="ab3comm"></a>`

Reply:
oh er I think its part of the UG by default? I didn't touch this.

The default one has stuff like
`<a name="features"></a>`
`<a name="commandsummary"></a>` etc
I'm not sure if it's needed but don't want to risk not grading or whatever.